tests/Aspire.NATS.Net.Tests/ConformanceTests.cs

```diff
@@ -14,7 +14,7 @@ public class ConformanceTests : ConformanceTests<INatsConnection, NatsClientSett
     private const string ConnectionSting = "nats://apire-host:4222";
```

Comment:
Side question, any chance you can update these tests to use https://www.nuget.org/packages/Testcontainers.Nats? (in a separate PR) That way the tests run in CI and we can ensure they pass successfully. See 

* https://github.com/dotnet/aspire/pull/2345
* https://github.com/dotnet/aspire/pull/2718
* https://github.com/dotnet/aspire/pull/2856

For examples.

Reply:
that was fairly easy. didn't know about testcontainers. thanks for that
* #2886